project/CLI.py

```diff
@@ -44,6 +53,15 @@ def test_login_bad_username(self):
     def test_login_bad_args(self):
         self.assertEqual("Invalid parameters", self.cli.command("login gamemaker"), "Invalid parameters")
 
+class TestCreateGame(unittest.TestCase):
+    def setUp(self):
+        self.cli = CLI()
+
+    def test_create_game_success(self):
+        self.assertEquals("Game Creation Success", self.cli.command("creategame"), "GameCreate message not correct")
```

Comment:
We do not check it now, but you will need to log in as gamemaker before you can call creategame